ast/compile.go

```diff
@@ -1353,6 +1354,31 @@ func (c *Compiler) getExports() *util.HashMap {
 	return rules
 }
 
+func (c *Compiler) checkUnusedImports() {
```

Comment:
Could you please add a test for this? https://github.com/open-policy-agent/opa/blob/d7c17d167a80fe37b316a0e7d178ad73e8e43c59/ast/compile_test.go#L1503-L1540 might be a good example to follow.

Reply:
Will do!

Reply:
Added test cases for unused imports.